File: tests/fortran/intrinsics_extended/acos_one.f90
! vybe-test: fortran/intrinsics_extended/acos_one
! origin: languages/fortran/tests/fortran/test_intrinsics_extended.rs
program t
real :: x
x = acos(1.0)
if ((x) /= 0) then
    print *, "FAIL: want [0] got [", x, "]"
    stop 1
end if
end program t

File: tests/lua/math_trig_advanced/acos_one.lua
-- vybe-test: lua/math_trig_advanced/acos_one
-- origin: languages/lua/tests/lua/test_math_trig_advanced.rs

local __w1 = "true"
local __i = 0

do local __t = tostring(math.abs(math.acos(1)) < 1e-10); __i = __i + 1
  if __i == 1 and __t ~= __w1 then error("FAIL: want [" .. __w1 .. "] got [" .. __t .. "]") end end

if __i == 0 then error("FAIL: no output, wanted [" .. __w1 .. "]") end
